### ISSUE TITLE
Media Browser: icons and buttons are shown incorrectly for viewports with 1280px width

### ISSUE THREAD

**Issue by FlorianSmura** [neos/neos-development-collection#3417]
### Description

When working with a laptop, which has a viewport of 1280px, the media browser shows an incorrect button style:
![grafik](https://user-images.githubusercontent.com/73702758/130081834-046a6406-8e67-4785-8b9c-c1e7f41ef5dd.png)

### Steps to Reproduce

1. Navigate to a page, which has a NoteType containing an image option
2. Open the Media Browser by clicking the "Media" button
3. Make sure to change the viewport to 1280px width via Browser DevTools

#### Expected behavior

Media Browser Menu is displayed as with >1280px viewport width.
![grafik](https://user-images.githubusercontent.com/73702758/130192011-c2531e0e-556f-474a-ac1e-62a9f8a022ca.png)

#### Actual behavior

NEOS icon is white, button color is white, so the visibility is broken.
From a UX point of view it's not clear which media source/collection/tag is actually selected.

### Affected Versions

Neos: 5.3

Flow: 6.3

**Comment by markusguenther** [comment]
Tried that with the latest LTS version, and it is a more general issue.
So change the version to 5.3 for this bug.

![Screenshot 2021-08-20 at 13 34 11](https://user-images.githubusercontent.com/1014126/130227481-e205d244-38d3-4b58-98ec-d9cd9cfc8a3c.png)

**Comment by markusguenther** [comment]
I checked this and with some CSS tricks we can tweak this.

As we are not able to use the `fill` rule for the SVG, because we just don't know what kind of SVG file we will get from the asset source. And we also can not introduce a new configuration as bug fix for having a darker SVG icon as well.

So i played a bit and we can manipulate the white icons with a filter like that.
```
  a.neos-active .neos-media-assetsource-icon {
    filter: invert(.5) sepia(1) saturate(5) hue-rotate(175deg);
  }
```

<img width="746" alt="Screenshot 2021-09-24 at 12 16 44" src="https://user-images.githubusercontent.com/1014126/134659285-1f074233-a281-4958-b8cf-b1c27b346250.png">

**Comment by markusguenther** [comment]
Has been fixed and will be released with version 5.3.5, 7.0.4, 7.1.3 and the new 7.2 soonish
